src/components/Players.js

```diff
@@ -21,11 +23,20 @@ const Players = () => {
   return (
     <div className='players'>
       <h2>Online Players HERE</h2>
-      <ul>
+      <List>
         {onlineUsers.map((user) => (
-          <li key={user.id}>{user.username}</li>
+          <ListItemButton
+            key={user.id}
+            onClick={async () => {
+              const localUserId = getLocalUser().id
+              const invitedUserId = user.id
+              if (localUserId !== invitedUserId) await createGame([localUserId, invitedUserId])
```

Comment:
What will happen after createGame() is called? Navigation to the game screen or just mark the player as invited?

Reply:
Error handling might be nice to have, although not strictly required for the MVP.